kotlin.system.ProcessKt
    private static final java.lang.Void exitProcess(int)
    {
        int $i0;
        java.lang.RuntimeException r0;
        java.lang.Throwable r1;

        $i0 := @parameter0: int;

        staticinvoke <java.lang.System: void exit(int)>($i0);

        r0 = new java.lang.RuntimeException;

        specialinvoke r0.<java.lang.RuntimeException: void <init>(java.lang.String)>("System.exit returned normally, while it was supposed to halt JVM.");

        r1 = (java.lang.Throwable) r0;

        throw r1;
    }

